apps/src/aichat/views/demoMessages.ts

```diff
@@ -0,0 +1,81 @@
+// Will delete once we've connected UI to backend.
+const demoChatMessages = [
+  {
+    id: '1',
+    name: 'HistoryBot',
+    role: 'assistant' as const,
```

Comment:
@sanchitmalhotra126 - I received an error unless I assigned `role` as above (with `as const`) after I defined the `role` types as either `system`, `user`, or `assistant`. @bencodeorg - heads up.